servers/partner-ci/common.yaml

```diff
@@ -119,16 +115,17 @@
             FORCE_DISABLE_UPDATES=false;
             FUEL_STATS_ENABLED=false;
             GIT_CURL_VERBOSE=false;
+            ISO_STORAGE=/storage/downloads;
             MAKE_SNAPSHOT=true;
             NODES_COUNT=10;
             OPENSTACK_RELEASE=Ubuntu;
             SLAVE_NODE_CPU=4;
             SLAVE_NODE_MEMORY=4096;
             SNAPSHOT_ARTIFACTS_FILE=snapshots.params;
             SNAPSHOT_OUTPUT_FILE=extra_repos.sh;
+            VENV_PATH=$HOME/90-venv;
```

Comment:
90-venv is used for fuel90
100-venv is used for fuel100

VENV_PATH wasn't a constant